x-pack/plugins/synthetics/public/apps/synthetics/components/getting_started/simple_monitor_form.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFieldText,
+  EuiFormRow,
+  EuiForm,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useSimpleMonitor } from './use_simple_monitor';
+import { ServiceLocationsField } from './form_fields/service_locations';
+import { ConfigKey, ServiceLocations } from '../../../../../common/runtime_types';
+import { useFormWrapped } from '../../../../hooks/use_form_wrapped';
+
+export interface SimpleFormData {
+  urls: string;
+  locations: ServiceLocations;
+}
+
+export const SimpleMonitorForm = () => {
+  const {
+    control,
+    register,
+    handleSubmit,
+    formState: { errors, isValid, isSubmitted },
+  } = useFormWrapped({
+    mode: 'onBlur',
+    reValidateMode: 'onChange',
+    shouldFocusError: true,
+    defaultValues: { urls: '', locations: [] as ServiceLocations },
+  });
+
+  const [monitorData, setMonitorData] = useState<SimpleFormData | undefined>();
+
+  const onSubmit = (data: SimpleFormData) => {
+    setMonitorData(data);
+  };
+
+  const { loading } = useSimpleMonitor({ monitorData });
+
+  const hasURLError = !!errors?.[ConfigKey.URLS];
+
+  return (
+    <EuiForm onSubmit={handleSubmit(onSubmit)} component="form" isInvalid={isSubmitted && !isValid}>
```

Comment:
EuiForm neeeds `noValidate` prop to prevent native validation on submission. _Not a blocker though_.

<img width="604" alt="image" src="https://user-images.githubusercontent.com/2748376/169278772-de702ced-62fb-4e44-a340-f89b7bca40a0.png">